pom.xml

```
@@ -945,6 +945,17 @@ limitations under the License.
         <enabled>false</enabled>
       </snapshots>
     </repository>
+<!--    Added below repository as a temporary measure,
+        It can be removed once commons-email2-* dependencies are added to our repo-->
+    <repository>
+      <id>apache central</id>
+      <name>Central Repository</name>
+      <layout>default</layout>
+      <snapshots>
+        <enabled>true</enabled>
+      </snapshots>
```

Let's make sure that this will not used when trying to download non-snapshot artifacts
```suggestion
      <snapshots>
        <enabled>true</enabled>
      </snapshots>
      <releases>
        <enabled>false</enabled>
      </releases>
```

---

updated accordingly